src/NServiceBus.Persistence.AzureStorage/SagaPersisters/SecondaryIndices/PartitionRowKeyTuple.cs

```diff
@@ -61,15 +61,10 @@ public static PartitionRowKeyTuple Parse(string str)
             {
                 return null;
             }
-            var strings = str.Split(Separators, StringSplitOptions.None);
+            var strings = str.Split(new [] { Separator }, StringSplitOptions.None);
```

Comment:
The only reason for this was probably my prev work on some serializers and wanting to skip this one allocation. If this change make it more readable, I'm more than ok with it.